### PR TITLE
Feature avl

### PR DESCRIPTION
**Pull request by mclarke2**
Changes to the Aerodynamics and Stability AVL functions

### CODE REVIEW

**Comment by planes** [comment]
You have conflicts.

File: trunk/SUAVE/Analyses/Stability/AVL.py [hunk 1]
```diff
@@ -147,12 +137,10 @@ def initialize(self,conditions):
         mac                 = geometry.wings['main_wing'].chords.mean_aerodynamic        
         mach                = conditions.freestream.mach_number
         AoA                 = conditions.aerodynamics.angle_of_attack
```
Comment (timdmacdo) [inline trunk/SUAVE/Analyses/Stability/AVL.py]: I actually think it looks better with the spacing in this section. Some of these assignments can be logically separated.

Comment (mclarke2) [inline trunk/SUAVE/Analyses/Stability/AVL.py]: Ok I will make the relevant changes

Comment (planes) [inline trunk/SUAVE/Analyses/Stability/AVL.py]: Once things are grouped I am okay with pulling this in.

**Comment by timdmacdo** [comment]
lgtm

**Pull request merged**